README.md

```diff
@@ -23,6 +26,37 @@ This repository contains an SDK and Jupyter notebooks with examples to showcase
 - [Build Status](#BuildStatus)
 - [Additional References](#Refs)
 
+# <a name="intro"></a> 
+
+# Overview of Interpret-Community
+Interpret-community is a practical and community-driven repository that hosts a wide range of community-developed machine learning interpretability techniques. This repository makes it easy for anyone involved in the development of an AI system to achieve transparency around their machine learning models. Data scientists, machine learning engineers, and researchers can easily add their own interpretability techniques via the set of extension hooks built into the parent repository, Interpret, and expand this repository to include their custom-made interpretability techniques.
+
+
+This repo incorporates community-driven state-of-the-art interpretability solutions under one roof with a unified set of data structures and API. Users could experiment with different interpretability methods, and/or add their custom-made interpretability techniques and perform comparative analysis to evaluate their brand new explainers. Using these tools, one can explain machine learning models globally on all data, or locally on a specific data point using the state-of-art technologies in an easy-to-use and scalable fashion. In particular, this released open source toolkit: 
+1. Actively incorporates state-of-the-art experimental interpretability techniques, and allows for further expansion by researchers and data scientists 
+2. Creates a common API across the integrated libraries    
+3. Applies optimizations to make it feasible to run on real-world datasets at scale   
+4. Provides improvements such as the capability to "reverse the feature engineering pipeline" to provide users with feature importance values in terms of raw features
+5. Provides interactive and exploratory visualization to empower data scientists to get the most level of insights about their data
+# <a name="comparison"></a> 
+
+# Interpret vs. Interpret-Community
+
+
+Interpret-Community and its parent, Interpret, both serve as a one-stop-shop tool for researchers, machine learning engineers, software developers, data scientists, and business executives to get insights about the model predictions and why model made the prediction it did. The parent repository, Interpret, hosts the interpretability techniques that are widely accepted by research community and have been published in prestigous  publications and press (e.g., SHAP, EBM). Interpret-Community is an extension to Interpret and hosts experimental interpretability techniques that are yet to receive acceptance from the research community.            

```

Comment:
and why model made the prediction it did -> and why the model made the predictions it did

Reply:
by research community -> by the research community

Reply:
prestigous  publications has extra double-space

Reply:
hosts experimental interpretability techniques that are yet to receive acceptance from the research community and functionality for data scientists and software engineers to simplify their workflow